### ISSUE TITLE
Recaptcha & proxy configuration

### ISSUE THREAD

**Issue by kows** [tanveery/recaptcha-net#35]
With some extended testing on our different environments I came to the following conclusion regarding the usage of a proxy and this library.
It seems to me, by using WebRequest.GetSystemWebProxy() in VerifyRecaptchaResponse of RecaptchaVerificationHelper, the proxy that is used is always the proxy defined in settings of IE for current user (in our case NetworkService, defined on application pool).
Overriding this proxy with web.config fails because the result of method above always overrides the default proxy.
Currently I'm keeping my proxy on the NetworkService account but it would be more usefull if this is adaptable by config.

**Comment by weedkiller** [comment]
I have a similar issue where I want to give the clients during login a` check box Proxy true`, if they're behind a proxy

**Comment by woodpd** [comment]
I agree with kows, we really need to be able to define the proxy from config rather than just using GetSystemWebProxy() which can really cause deployment issues if you don't have the ability to set that. Everything should be configurable with the app and its a reasonably simple feature to add

**Comment by UweKeim** [comment]
We urgently need this, too. Any chance to ever implement setting an `IWebProxy` from outside?
